src/deroff.rs

```diff
@@ -1,352 +1,277 @@
+            "TP" => &"\u{00de}",
+            "AE" => &"\u{00c6}",
+            "ae" => &"\u{00e6}",
+            "OE" => &"OE",
+            "oe" => &"oe",
+            ":a" => &"\u{00e4}",
+            ":A" => &"\u{00c4}",
+            ":e" => &"\u{00eb}",
+            ":E" => &"\u{00cb}",
+            ":i" => &"\u{00ef}",
+            ":I" => &"\u{00cf}",
+            ":o" => &"\u{00f6}",
+            ":O" => &"\u{00d6}",
+            ":u" => &"\u{00fc}",
+            ":U" => &"\u{00dc}",
+            ":y" => &"\u{00ff}",
+            "ss" => &"\u{00df}",
+            "'A" => &"\u{00c1}",
+            "'E" => &"\u{00c9}",
+            "'I" => &"\u{00cd}",
+            "'O" => &"\u{00d3}",
+            "'U" => &"\u{00da}",
+            "'Y" => &"\u{00dd}",
+            "'a" => &"\u{00e1}",
+            "'e" => &"\u{00e9}",
+            "'i" => &"\u{00ed}",
+            "'o" => &"\u{00f3}",
+            "'u" => &"\u{00fa}",
+            "'y" => &"\u{00fd}",
+            "^A" => &"\u{00c2}",
+            "^E" => &"\u{00ca}",
+            "^I" => &"\u{00ce}",
+            "^O" => &"\u{00d4}",
+            "^U" => &"\u{00db}",
+            "^a" => &"\u{00e2}",
+            "^e" => &"\u{00ea}",
+            "^i" => &"\u{00ee}",
+            "^o" => &"\u{00f4}",
+            "^u" => &"\u{00fb}",
+            "`A" => &"\u{00c0}",
+            "`E" => &"\u{00c8}",
+            "`I" => &"\u{00cc}",
+            "`O" => &"\u{00d2}",
+            "`U" => &"\u{00d9}",
+            "`a" => &"\u{00e0}",
+            "`e" => &"\u{00e8}",
+            "`i" => &"\u{00ec}",
+            "`o" => &"\u{00f2}",
+            "`u" => &"\u{00f9}",
+            "~A" => &"\u{00c3}",
+            "~N" => &"\u{00d1}",
+            "~O" => &"\u{00d5}",
+            "~a" => &"\u{00e3}",
+            "~n" => &"\u{00f1}",
+            "~o" => &"\u{00f5}",
+            ",C" => &"\u{00c7}",
+            ",c" => &"\u{00e7}",
+            "/l" => &"/l",
+            "/L" => &"/L",
+            "/o" => &"\u{00f8}",
+            "/O" => &"\u{00d8}",
+            "oA" => &"\u{00c5}",
+            "oa" => &"\u{00e5}",
+
+            // Ligatures
+            "fi" => &"fi",
+            "ff" => &"ff",
+            "fl" => &"fl",
+            "Fi" => &"ffi",
+            "Ff" => &"fff",
+            "Fl" => &"ffl",
+            _ => return None,
+        })
+    }
+
+    // Much like the above, return small strings for now until we know what
+    // we might actually want to do
+    fn g_specs(key: &str) -> Option<&'static str> {
+        Some(match key {
+            "mi" => &"-",
+            "en" => &"-",
+            "hy" => &"-",
+            "em" => &"--",
+            "lq" => &"“",
+            "rq" => &"”",
+            "Bq" => &",,",
+            "oq" => &"`",
+            "cq" => &"'",
+            "aq" => &"'",
+            "dq" => &"\"",
+            "or" => &"|",
+            "at" => &"@",
+            "sh" => &"#",
+            "Eu" => &"\u{00a4}",
```

Comment:
Hmm. Might either need to open a separate issue. My assumption would be that this should be `€`, but in Rust that would be `"\u{20ac}"`. (In the interest of the first port being naïve, an issue to revisit seems fine.)

Reply:
I've gone ahead and used the Euro symbol, but opened https://github.com/rust-dc/fish-manpage-completions/issues/16 for us to follow up later.